modules/shared/components/atoms/VoteImage/VoteImage.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ReactElement, FC } from 'react';
+import React from 'react';
+import type { IVoteImage } from './IVoteImage';
+import OutlinedHeartIcon from '../../icons/outlinedHeart.svg';
+import FilledHeartIcon from '../../icons/filledHeart.svg';
+import OutlinedDislikeIcon from '../../icons/outlinedDislike.svg';
+import OutlinedLikeIcon from '../../icons/outlinedLike.svg';
+import FilledLikeIcon from '../../icons/filledLike.svg';
+import styles from './VoteImage.module.css';
+
+const VoteImage: FC<IVoteImage.IProps> = ({ oneImage }): ReactElement => {
+  return (
+    <div className="relative">
+      <div>
+        {oneImage ? (
+          <button type="button" className={`${styles.love} group`}>
+            <OutlinedHeartIcon className="group-hover:hidden" />
+            <FilledHeartIcon className="group-hover:block hidden" />
+          </button>
+        ) : (
+          <div className="flex">
+            <button type="button" className={`${styles.dislike} group`}>
+              <OutlinedDislikeIcon />
+            </button>
+            <button type="button" className={`${styles.like} group`}>
+              <OutlinedLikeIcon className="group-hover:hidden" />
+              <FilledLikeIcon className="group-hover:block hidden" />
+            </button>
+          </div>
```

Comment:
- while clicking on the `Like` icon it is behaviors as shown bellow 😂. `dancing like

https://user-images.githubusercontent.com/8650435/120899949-83fd1d80-c632-11eb-9bda-c55203ae6c3d.mov

Reply:
Ali told me about this and suggested a way to solve it already.